scripts/autogen

```diff
@@ -2363,11 +2364,86 @@ def gen_test_configs():
         gen_test_config(config_spec["path"], config_spec, default_config)
 
 
+def extract_bytecode_from_output(output_text):
```

Comment:
Please place those functions in the same place as they are in the mlkem-native autogen script, i.e., below`gen_citations` - that makes syncing these files much easier.

Reply:
@jakemas, general note: I think we try to let comments be resolved by the person who raised them, so we can be sure they're happy. So let's leave this to @mkannwischer.